g2p_registry_rest_api/__manifest__.py

```diff
@@ -20,6 +20,7 @@
         "extendable",
         "g2p_registry_group",
         "g2p_registry_individual",
+        "g2p_registry_addl_info",
```

Comment:
I'd not add this as a dependency because not everyone will use this module. I'd instead add it to another app that depends on `g2p_registry_rest_api` and `g2p_registry_addl_info`.
We use `pydantic` to allow this.

Reply:
I agree. I will reorder this in the next commit. Thanks.

Reply:
Although I didn't get your point about using pydantic.

Reply:
We use Pydantic to make it easier to have other apps extend the base REST API. So if for your needs you require to add fields, for example, this is possible. Take a look at the sample in OCA rest repo.